openapi-validator/src/main/java/org/ballerinalang/openapi/validator/ServiceValidator.java

```diff
@@ -186,18 +299,101 @@ public static void validateResource(OpenAPI openApi,
+                    Path relativePath;
+                    if (Paths.get(openapiPath.toString()).isAbsolute()) {
+                        relativePath = Paths.get(openapiPath.toString());
+                    } else {
+                        File file = new File(sourceRoot.toString());
+                        File parentFolder = new File(file.getParent());
+                        File openapiContract = new File(parentFolder, openapiPath.toString());
+                        relativePath = Paths.get(openapiContract.getCanonicalPath());
+                    }
+                    if (Files.exists(relativePath)) {
+                        try {
+                            openAPI = ServiceValidator.parseOpenAPIFile(relativePath.toString());
+                        } catch (OpenApiValidatorException e) {
+                            DiagnosticInfo diagnosticInfo = new DiagnosticInfo("BAL_OPENAPI_VALIDATOR_0019",
+                                    e.getMessage(), DiagnosticSeverity.ERROR);
+                            Diagnostic diagnostic = DiagnosticFactory.createDiagnostic(diagnosticInfo
+                                    , annotationNode.location());
+                            validations.add(diagnostic);
+                        }
+                    } else {
+                        String[] error = ErrorMessages.contactFileMissinginPath();
+                        DiagnosticInfo diagnosticInfo = new DiagnosticInfo(error[0], error[1],
+                                DiagnosticSeverity.ERROR);
+                        Diagnostic diagnostic = DiagnosticFactory.createDiagnostic(diagnosticInfo,
+                                fieldNode.location());
+                        validations.add(diagnostic);
+                    }
+                } else if (specificFieldNode.fieldName().toString().trim().equals("failOnErrors")) {
+                    String failOnErrors = openAPIAnnotation.toString();
+                    if (failOnErrors.trim().equals("false")) {
+                        kind = DiagnosticSeverity.WARNING;
+                        filters.setKind(kind);
+                    }
+                } else if (openAPIAnnotation instanceof ListConstructorExpressionNode) {
+                    ListConstructorExpressionNode list = (ListConstructorExpressionNode) openAPIAnnotation;
+                    String attributeName = specificFieldNode.fieldName().toString().trim();
+                    switch (attributeName) {
+                        case "tags":
+                            List<String> tags = setFilters(list);
+                            filters.setTag(tags);
+                            break;
+                        case "excludeTags":
+                            List<String> eTags = setFilters(list);
+                            filters.setExcludeTag(eTags);
+                            break;
+                        case "operatoins":
+                        case "excludeOperations":
```

Comment:
Why don't we consider these switch cases as default case?

Reply:
yeah, we can set this as the default case.